clip_retrieval/clip_filter.py

```diff
@@ -4,7 +4,7 @@
 import fire
 
 
-def clip_filter(query, output_folder, indice_folder, num_results=100, threshold=None):
+def clip_filter(query,output_folder, indice_folder, metadata_folder=None, clip_model="ViT-B/32",num_results=100, threshold=None):
```

Comment:
With the recent changes to include open clip, this may need to be updated to consider that.
I see that there is a separate file load_model.py which is not being reused everywhere, if that intention of that is to be reused, we can change this to use that instead, which is handling open clip already from what I can see.